### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by willkg**
Minor fixes to documentation and infrastructure stuff.

Do the docs look right? If not, then it's probably the case I couldn't figure out how things work.

r?

### CODE REVIEW

**Comment by willkg** [comment]
^^^ Two more.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+     DJANGO_CONFIGURATION=Dev
+
+   in your environment.
+
+
+Run tests
+=========
+
+1. run ``DJANGO_CONFIGURATION=Dev ./manage.py test`` to run Python tests
```
Comment (mythmon) [inline README.rst]: Is there any compelling reason to use RST? I have trouble speaking RST, so if we could stick with Markdown, that would make me happier. But if there is a good reason to go with it, we can do that.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+:author:  League of Assassins
+
+
+Install for hacking
+===================
+
+1. create a virtual environment
+2. run ``pip install -r requirements.txt``
+3. run ``DJANGO_CONFIGURATION=Dev ./manage.py migrate``
+4. run ``npm install``
```
Comment (mythmon) [inline README.rst]: It doesn't matter what order these come in, but I'd put the npm install as step 3, so the first few steps are all about getting dependencies installed.

**Comment by mythmon** [comment]
The docs are correct, and your changes look good (modulo the couple of notes about the README). This also makes me want to put a pre-commit linter in here really fast.

File: README.rst (hunk 1 above)
Comment (willkg) [inline README.rst]: reST is a lot more flexible than Markdown is, so I do everything in reST. However, if you prefer Markdown, that's fine. I'll switch it to Markdown.

**Comment by willkg** [comment]
^^^ Redid the README in Markdown.

**Comment by mythmon** [comment]
Thanks!

**Pull request merged**